types/setting.go

```diff
@@ -344,6 +348,28 @@ var (
 		ReadOnly:    false,
 		Default:     "10",
 	}
+	SettingDefinitionVolumeAttachmentDeletion = SettingDefinition{
+		DisplayName: "Volume Attachment Deletion Behavior",
+		Description: "Controls when Longhorn should delete a volume attachment on a failed node",
+		Category:    SettingCategoryDangerZone,
+		Type:        SettingTypeString,
+		Required:    true,
+		ReadOnly:    false,
+		Default:     string(VolumeAttachmentDeletionWait),
+		Options: []string{
+			string(VolumeAttachmentDeletionNever),
+			string(VolumeAttachmentDeletionWait),
+			string(VolumeAttachmentDeletionImmediate),
+		},
+	}
+)
+
+type VolumeAttachmentDeletionOption string
+
+const (
+	VolumeAttachmentDeletionNever     = VolumeAttachmentDeletionOption("never") // kubernetes default behavior
```

Comment:
`conservative`, `default`, `aggressive`

Reply:
I like the `never, wait, immediate` since it's more specific and documents the behavior.
I changed it to your suggested values.

Reply:
Don't use different names externally and internally if possible.

OK, I am fine with `never/wait/immediate`. Need to add more explanation on what does each option mean in the description.